main.js

```diff
@@ -204,8 +204,11 @@ class MyVbus extends utils.Adapter {
             }
 
             ctx.connection.on('connectionState', (connectionState) => {
-                this.log.info('Connection state changed to ' + connectionState);
-                if (connectionState !== 'CONNECTED') {
+                this.log.debug('Connection state changed to ' + connectionState);
+                if (connectionState === 'CONNECTED') {
+                    this.log.info('The Adapter is now connected to your Resol vBus hardware.');
```

Comment:
Hiermit probiere ich nur mal die GitHub Möglicheiten aus.
Ich bevorzuge kurze knackige log messages, z.B. "Connection to Resol VBus device established."
oder noch kürzer: "Connection established."
ach ja, die Resol-Schreibweise ist tatsächlich VBus (nicht vBus)

Reply:
Verstehe ich. Da solche Meldungen aber  nicht alleine für die Entwickler sonden vor allem für die Nutzer sind, bevorzuge ich kurz, knackig und vor allem präzise zu sagen was tatsächlich Fakt ist und nicht Verallegmeinerungen zu nutzen die Interpretationsspielraum lassen nur weil sie schön kurz sind.
```suggestion
                 if (connectionState === 'CONNECTED') {
                    this.log.info('Local connection to VBus hardware established.');
```
Das lässt offen ob es sich um ein Resol Device oder Eigenbau handelt, schließt eine Web-API aber aus.

Reply:
Hmm, Es handelt sich nicht generell um eine lokale Verbindung; es kann auch eine remote Verbindung über VBus.net aufgebaut werden; daher der Vorschlag: Connection to Resol VBus device established.
Begründung: Eigenbau würde ich erstmal nicht im Auge haben. 
Resol versteht unter devices die Verbindungsgeräte (DL3, KM2..) mit den implizit daran angeschlossenen Reglern.  myvbus Verbindungen sind immer**) VBus-Protokoll-Verbindungen über die Verbindungsgeräte bis zu den Reglern. Diese Verbindung kann  lokal im LAN  (IP, port 7053) oder oder seriell (com/tty) oder remote über VBus.net (URL, Via, port7053) erfolgen .  VBus.net funktioniert hier wie ein Router. Daher würde ich auf den Begriff "Local" verzichten.
**) myvbus kann Stand heute nicht das Web-API(json-based) von DL2/3  ansprechen.

Reply:
Dann ist das wohl so.

```suggestion
                    this.log.info('Connection established');
```